lightningd/options.c

```diff
@@ -191,6 +191,38 @@ static char *opt_add_addr(const char *arg, struct lightningd *ld)
 	return opt_add_addr_withtype(arg, ld, ADDR_LISTEN_AND_ANNOUNCE, true);
 }
 
+static char *opt_alt_subdaemon(const char *arg, struct lightningd *ld)
+{
+	// example arg: "hsmd:rmthsmd"
```

Comment:
style nit: We use `/* */`

Reply:
also, shouldn't it be `lightning_hsmd:rmthsmd`?

Reply:
fixed both